src/index.html

```diff
@@ -1,13 +1,152 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
+
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Stars</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Stars Blocks</title>
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div
+        class="stars__star">
```

Comment:
if you have just 1 attribute it should be in the same line (same with every <div> below)
you can use prettier to format code properly